types/types.go

```diff
@@ -145,6 +147,10 @@ type ImageDestination interface {
 	// ShouldCompressLayers returns true iff it is desirable to compress layer blobs written to this destination.
 	ShouldCompressLayers() bool
 
+	// AcceptsForeignLayerURLs returns true iff foreign layers in manifest should be actually
+	// uploaded to the image destination, false otherwise.
+	AcceptsForeignLayerURLs() bool
```

Comment:
I’m sorry, I have suggested this exact wording and this is entirely my fault, but `AcceptsForeignLayerURLs` is the opposite of what the flag means: `false` = keep the `BlobInfo` with the URL, `true` = make a copy.

(Writing this out in case I am making another mistake.)